sw/device/lib/dif/dif_rstmgr.c

```diff
@@ -21,7 +21,7 @@
 RSTMGR_RESET_INFO_CHECK(Por, INFO_POR);
 RSTMGR_RESET_INFO_CHECK(LowPowerExit, INFO_LOW_POWER_EXIT);
 RSTMGR_RESET_INFO_CHECK(Ndm, INFO_NDM_RESET);
-RSTMGR_RESET_INFO_CHECK(HwReq, INFO_HW_REQ);
+//RSTMGR_RESET_INFO_CHECK(HwReq, INFO_HW_REQ);
```

Comment:
@silvestrst 
so the HW field now has multiple bits and has become `FIELD` or `OFFSET`.
But this obviously doesn't fit too well with the existing assert check.  Any suggestions on what would be the cleanest fix?

Reply:
I think the easiest way to fix it would be:
```
dif_rstmgr.c:
_Static_assert(kDifRstmgrResetInfoHwReq ==
               (RSTMGR_RESET_INFO_HW_REQ_MASK <<
                RSTMGR_RESET_INFO_HW_REQ_OFFSET),
               "kDifRstmgrResetInfoHwReq must match the register definition!");

dif_rstmgr.h:
kDifRstmgrResetInfoHwReq = (0x3 << 3),
```

I think in light of there being multiple TOP levels - the static assertions like that will need to go away, but this is a problem to sort for later.

Reply:
thanks! updated per your suggestions

Reply:
had to update a few other things, would be great if you can have a look.